pkg/pidgin-telegram-purple

```diff
@@ -0,0 +1,24 @@
+[deps]
+pidgin
+libgcrypt
+libwebp
+gettext
+zlib
+git
+
+[build]
+git clone --recursive https://github.com/majn/telegram-purple
```

Comment:
I know this is not good, but the alternative is hosting a properly packed tarball ourselves.